src/utils/process-csv.js

```diff
@@ -46,20 +46,23 @@ function processNumOriginationsData( csv, group ) {
   csv = Papa.parse( csv ).data;
   csv.shift();
   csv.forEach( function( dataPoint ) {
-    var arr = [];
-    var series = dataPoint[2];
-    arr.push( formatDate( dataPoint[0] ) );
-    arr.push( parseFloat( dataPoint[1] ) );
 
-    if ( group ) {
-      series = dataPoint[3];
-    }
+    if ( _dateCategory( dataPoint[0] ) > new Date( '2009-01-01 00:00:00 UTC' ) ) {
```

Comment:
This date format causes problems with IE and Firefox. Try this format: https://github.com/cfpb/cfpb-chart-builder/pull/23/files